hydra-access-controls/hydra-access-controls.gemspec

```diff
@@ -19,9 +19,9 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '>= 1.9.3'
 
   gem.add_dependency 'activesupport', '~> 4.0'
-  gem.add_dependency "active-fedora", '~> 9.13'
+  gem.add_dependency "active-fedora", '~> 10.0.0.beta1'
```

Comment:
Do we need to be this pessimistic, or could we make this '>= 10.0.0.beta1, < 11'?